_posts/2022-12-21-python-shorts-opensgid-pandas.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Python Shorts: Loading an Open SGID Layer into pandas"
+author:
+  display_name: Jake Adams and Scott Davis
+  email: jdadams@utah.gov
+date: 2022-12-22 11:00:00
+categories:
+  - Featured
+  - Developer
+tags:
+  - python
+---
+
+There are a [myriad of ways]({% link sgid/open-sgid/index.md %}) you can access data from the Open SGID. However, these all require some form of desktop client application. What if you want programmatic access to our spatial data? Because the Open SGID uses the open-source PostGIS extension to Postgres, you can use any libraries that can load Postgres tables and process the geometry data (or just ignore that if all you want is the tabular data).
+
+In the python world, pandas DataFrames are a common and powerful structure for evaluating data. Our developer team has put together this code snippet for loading an SGID layer into either a geopandas geodataframe or an Esri spatially-enabled dataframe.
```

Comment:
**suggestion**: linking to the docs pages on these to dataframes would be a nice touch.

Reply:
Great idea. Donezo.